ios/src/TangramMap/TGHttpHandler.h

```diff
@@ -0,0 +1,78 @@
+//
+//  TGHttpHandler.h
+//  tangram
+//
+//  Created by Karim Naaji on 11/23/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ A network request completion callback, called when a download request of `TGHttpHandler`
+ completed an asynchronous request.
+*/
+typedef void(^TGDownloadCompletionHandler)(NSData*, NSURLResponse*, NSError*);
+
+/**
+ A configurable http handler used in a `TGMapViewController`.
+
+ `TGMapViewController` provides a default http handler with the following configuration:
+
+    - cache location: `/tangram_cache`
```

Comment:
We set the default to `tangram_cache` [here](https://github.com/tangrams/tangram-es/blob/master/ios/src/TGMapViewController.mm#L748).